XavierGuertin/WildfireVisualizationProject
# #29 feat(frontend) - Added the ability to switch between different map layers and to draw bounds for wildfire events.

### Summary
Added the ability to switch between different map layers and to draw bounds for wildfire events.

### Changes Made
- Restructured the way that components are created and handled in the frontend 
- Removed placeholder UI elements
- Added three layers (default, topographical, satellite)
- Added a mockup of how we could render wildfire areas on the map
- Retrieved maps from MapTiler to fix localization and to get additional map types

### Testing Instructions
1. Click the "Views" button (bottom-left) to expand the sidebar, showing three clickable layer images.
2. Expand the sidebar, click each layer image (default, topographical, satellite), and confirm the map updates correctly for each selection.
3. Click again to collapse back to the icon.

### Screenshots (if applicable)
![image](https://github.com/user-attachments/assets/9642faeb-7255-492b-8e0b-8a0741d8e1d7)
![image](https://github.com/user-attachments/assets/d5047ce3-356c-4e05-8e8e-259859fbc6f1)
![image](https://github.com/user-attachments/assets/26683a44-8935-4150-bab3-dbac14779e3e)


https://github.com/user-attachments/assets/7f9d78fd-30f1-4467-86cc-4fc5711dc883



### Additional Notes
The map was made using OpenLayers for rendering and managing map layers.
